### PR TITLE
completed JS/if

### PR DESCRIPTION
**Pull request by Quintin-Russell**
image

### CODE REVIEW

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    }
+  } else {
+    document.write('invalid pH level');
+  }
+}
+
+function introduceWarnerBro(name) {
+  if ((name === 'yakko') || (name === 'wakko') || (name === 'dot')) {
+    if ((name === 'yakko') || (name === 'wakko')) {
+      document.write("We're the warner brothers!");
```
Comment (BrettAlbright) [inline javascript-if/main.js]: all of these should be return statements

**Review by Zhiwutian — APPROVED** [review]
just update with returns and you are good!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
sorry, hit wrong button!

File: javascript-if/main.js (hunk 1 above)
Comment (Quintin-Russell) [reply javascript-if/main.js]: ahhh that was i was asking about

**Comment by Quintin-Russell** [comment]
ready for review

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**